components/scream/src/share/tests/workspace_tests.cpp

```diff
@@ -175,7 +171,7 @@ static void unittest_workspace()
 #ifdef WS_EXPENSIVE_TEST
     if (true)
 #else
-    if (omp_get_max_threads() == 2) // the test below is expensive, we don't want all threads sweeps to run it
+    if ( ExeSpace::concurrency() ) // the test below is expensive, we don't want all threads sweeps to run it
```

Comment:
Looks like you forgot == 2

Reply:
ahhhh yes, thanks for catching that :-0

Reply:
Should it actually be `<=2`? We want the single thread test to go through it too, no?

Reply:
@bartgol , we only want to run this expensive test once and we want that run to be multithreaded.

Reply:
Ah, ok. Thanks. Btw, I can't find any place in the repo where that macro is defined, not even in the machine files or the test scripts. Is that something one must remember to add manually if need be?

Reply:
@bartgol , that's right, it can only be turned on manually.